packages/web-components/fast-components/src/listbox/listbox.styles.ts

```diff
@@ -20,16 +23,40 @@ import {
 } from "../design-tokens";
 import { heightNumber } from "../styles/size";
 
+/**
+ * A reflection of the size attribute.
+ *
+ * @public
+ */
+export const listboxSize = DesignToken.create<number>({
+    name: "listbox-size",
+    cssCustomPropertyName: "size",
+});
+
+export const listboxStrokeOffset = DesignToken.create<number>(
```

Comment:
Is this the gap between the list options and the edge of the box? If so I'd prefer to wait for the aforementioned density work as well as this will change how that padding is set. https://github.com/microsoft/fast/pull/5511

Reply:
We're not holding for density in this. We need to ship multiselect. The density work will be breaking, I'm not sure how holding on this moves us ahead.